db/migrate/20170118010335_create_ipeds_ics.rb

```diff
@@ -0,0 +1,125 @@
+class CreateIpedsIcs < ActiveRecord::Migration
+  def change
+    create_table :ipeds_ics do |t|
+      # Used in the building of DataCsv
+      t.string :cross, null: false
+      t.boolean :credit_for_mil_training
+      t.boolean :vet_poc
+      t.boolean :student_vet_grp_ipeds
+      t.boolean :soc_member
+      t.string :calendar
+      t.boolean :online_all
+      t.integer :calsys, null: false
+      t.integer :distnced, null: false
+      t.integer :vet2, null: false
+      t.integer :vet3, null: false
+      t.integer :vet4, null: false
+      t.integer :vet5, null: false
+
+      # Not used in building DataCsv, but used in exporting source csv
+      t.integer :peo1istr
```

Comment:
all of these t.integers, is it OK if they end up being null? or should you have null: false on these as well?